source/mainnet/smart-contracts/onboarding-guide-solana-developers/faq.rst

```diff
@@ -0,0 +1,765 @@
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAccountAddress>
+
+    If you insert your account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDAz9WdEy
+
+    The above request will return a transaction hash which you can look up on the block explorer.
+    You can only request CCD a single time for each account address via this API endpoint.
+    If you already submitted a request before, it will be an old transaction hash that is returned.
+
+    .. image:: ./images/onboarding_ethereum_developers_3.png
+        :width: 100 %
+
+    **Option 3:**
+    Alternatively, you can use `Postman <https://www.postman.com/>`_ and insert the following data before clicking the `Send` button.
+
+    .. image:: ./images/onboarding_ethereum_developers_4.png
+        :width: 100 %
+
+    **Option 4:**
+    If you have the curl package and the ``concordium-client`` tool installed on your Unix-like operating systems, you can request CCD to any of your alias account addresses.
+    If you already sent a previous request to the wallet proxy, you can not request any more CCD to the same account address.
+    Look up one of your alias account addresses instead and use it for your request.
+    The CCD will be credited to your canonical account address.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias <YourAccountAddress> --alias <number>
+
+    If you insert your account address and a number correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $concordium-client account show-alias 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j --alias 17
+
+    This generates the output:
+
+    .. code-block:: console
+
+        The requested alias for address 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JaqovViXS4j is 4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    Copy your alias address to the below command.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/<YourAliasAccountAddress>
+
+    If you insert your alias account address correctly, the command should look similar to the below line.
+
+    .. code-block:: console
+
+        $curl -X PUT https://wallet-proxy.testnet.concordium.com/v0/testnetGTUDrop/4phD1qaS3U1nLrzJcgYyiPq1k8aV1wAjTjYVPE3JXBDCpCaUT6
+
+    **Option 5:** If you need plenty of CCD for large-scale testing,
+    please contact Concordium’s technical support via support@concordium.software.
+
+.. dropdown::  Is there something similar to transaction fees?
+
+    Yes, transaction fees are called NRG (pronounced energy) on the Concordium chain.
+    The block limit is 3 million NRG. Transaction fees are fixed in Euro and are much
+    cheaper than on Solana.
```

Comment:
Transaction fees on Concordium are much cheaper than on Ethereum (do we know if it is also much cheaper than on Solana - I am not familiar enough with Solana)

Reply:
Good point! I don't know much about this either. Maybe @chrmatt could comment on this, but it's vacation time.
I'll remove the comparison from this section fo now.

Reply:
@dg-concordium I removed the part comparing Solana fees to Concordium. You're welcome to put it back if it was actually accurate :)

Reply:
Fees on Solana are very cheap, I think, even cheaper than the 1 Cent we have. What is unique about Concordium is that the fees are converted from NRG to CCD using a recent exchange rate to keep the prices fixed in EUR terms.

Reply:
@chrmatt.  thanks for your answer!

Would it be correct to write something like the following?

"Transaction fees are fixed in Euro and are currently set to 1 Eurocent. Fixing transaction fees in Euro is a unique feature of Concordium that provides predictable transaction costs."

Reply:
Resolved after comments by @chrmatt.